docs/changelog.md

```diff
@@ -55,8 +55,14 @@ description: |
     combination with [`layout`].
   - The height of a `block`, `image`, `rect`, `square`, `ellipse`, or `circle`
     can now be specified in [fractional units]($fraction)
-  - The [`scale`] function now supports non-relative lengths for `x` and `y`.
-    This way an element of unknown size can be scaled to a fixed size.
+    <!--
+        Add "for example/use case see" and link some `fraction` example,
```

Comment:
fr is already linked. the changelog isn't the place for examples. we can add more explanation to the fraction docs, but that's out of scope here

Reply:
Out of scope for a fixup/touch-up PR. You can open an issue if you want (the issue counter though 🥲)

Reply:
> Where exactly?

Wait, I've read the wrong thing. I was looking below as if it was doc string. I see the link now.

> the issue counter though

Oh, haha, I only understood this after a couple of tries. Yeah, since around 2024-01-01 it increased by more than 100 (soon will be 600 🎉). But I noticed that there are a lot of easy ones that were thankfully labeled as such. I'm looking forward to eliminating those in the near future (hopefully in this or next month).